jaxrs-spec/src/main/asciidoc/chapters/providers/_lifecycle_and_environment.adoc

```diff
@@ -14,11 +14,10 @@
 By default a single instance of each provider class is instantiated for
 each JAX-RS application. First the constructor (see
 <<provider_class_constructor>>) is called, then any requested dependencies
-are injected (see Chapter <<context>>), then the appropriate provider
-methods may be called multiple times (simultaneously), and finally the
-object is made available for garbage collection.
-<<providercontext>> describes how a provider obtains access to other
-providers via dependency injection.
+are injected, then the appropriate provider methods may be called multiple
+times (simultaneously), and finally the object is made available for
+garbage collection. <<providercontext>> describes how a provider obtains
+access to other providers via dependency injection.
```

Comment:
Perhaps all this paragraph could be replaced by simply stating that a Jakarta REST provider is in application scope (as defined by CDI) and instantiated and initialized accordingly. WDYT?

Reply:
By default in application scope, for the case someone would want to use some other scope, or singleton for user-provided instances (`Application.getSingletons()`)

Reply:
@jansupol Side note:  Application.getSingletons() was deprecated in 3.1.  Should we consider removing it in 4.0?

Reply:
@jim-krueger I think we should remove it. There's no need to confuse developers with the notion of a "singleton" when we now can take advantage of CDI's application scope.